static-site/docs.udon

```diff
@@ -1,13 +1,55 @@
+
+## [Getting Started](/docs/getting-started)
+
+If you'd like to install Arvo, get an Azimuth point, and/or get on the Urbit network, you'll find guides here. Dive in, and don't be afraid to [ask for help](mailto:support@urbit.org)!
+
+The three pages below are meant to be followed in order.
+
+- The [Bridge section](/docs/getting-started/azimuth) is the first step. It tells you how to access your Azimuth identity using our client.
+- The [guide to installing Urbit](/docs/getting-started/installing-urbit) is the second step. It tells you how to get our software on your machine.
+- The [guide to booting a ship](/docs/getting-started/booting-a-ship) is the third and final step. It walks you through using what you did in the previous steps to get onto the Arvo network.
+
+In addition to those primary pages, the documents below are helpful but optional for using Urbit.
+
+- [Creating a Development Ship](/docs/getting-started/creating-a-development-ship), a walkthrough for creating an unnetworked a a platform for coding.
+- [Operating a Star](/docs/getting-started/creating-a-development-ship)
+
+## [Concepts](/docs/concepts)
+
+If you're looking for a high-level introduction to the Urbit project and its components, start here.
+
+- [Arvo vs. Azimuth](/docs/introduction/arvo-vs-azimuth), a quick explanation of the two parallel systems that compose Urbit.
+- [Technical Overview](/docs/introduction/technical-overview), a more in-depth description of the entire Urbit stack.
+- [Contributing](/docs/introduction/contributing), a guide to getting involved with the Urbit project.
+- [Community Tutorials](/docs/introduction/community-tutorials), a collection of unofficial but useful Urbit learning resources.
+- [Source Code](/docs/introduction/source-code-overview), an explanation of Urbit's constituent GitHub repos.
+- [Galaxies, Stars, and Planets](/docs/introduction/galaxies-stars-and-planets), an explanation of the Urbit address-space hierarchy.
+
+## [Learn](/docs/learn)
+
+If you're interested in contributing to the project, building applications, or understanding the system technically, take a look at our tutorials and explanations of the Urbit components.
+
+- [The Hoon Tutorial](/docs/learn/arvo/hoon), for our programming language. To learn the system, start here.
+- [The Arvo section](/docs/learn/arvo), for the internals of Arvo, the Urbit operating system.
+
+## [Reference](/docs/reference)
+
+If you're a developer looking for Arvo reference documentation, this section is for you.
+
+- [The Glossary](/docs/reference/glossary) will help if you're confused by all the new jargon.
+- [The Cheat Sheet](/docs/reference/cheat-sheet) is a compact document for looking up Hoon expressions.
+- [The Hoon Expressions](/docs/reference/hoon-expressions) section contains more comprehensive reference material for Hoon expressions.
+- [The Standard Library](/docs/reference/library) section documents the Hoon standard library.
```

Comment:
I wouldn't make the "The" part of the links in the above bullets, so i.e. `- The [Standard Library](/docs/...)`